### DISCUSSION TITLE
So how is everyone's experience with this project? Compared to cd is it looking more time saving,fast and beneficial?

### DISCUSSION THREAD

**Discussion by Greatz08** [ajeetdsouza/zoxide#746] [General]
Just saw this project recently before testing just wanted to ask for some honest review from everyone who is using this.Hope to get some valuable response :-)).Whatever you feel about this do comment for it

**Comment by ajeetdsouza** [comment]
My (probably biased) advice:

- zoxide can function as a drop-in replacement for the basic `cd` command if you use `zoxide init --cmd cd`
- It only takes a few minutes to set up
- It's easy enough to uninstall (just remove the binary!), so there's not much to lose

It saves me a tonne of time personally, and makes my terminal more fun to use 😄

**Reply by edvardm** [reply]
I feel like I can't almost use computer without it. It's amazing, especially combined with interactive selection (zi)
